locksmith/src/routes/v2/verifier.ts

```diff
@@ -0,0 +1,30 @@
+import express from 'express'
+import networks from '@unlock-protocol/networks'
+import { Web3Service } from '@unlock-protocol/unlock-js'
+import { VerifierController } from '../../controllers/v2/verifierController'
+import { lockManagerMiddleware } from '../../utils/lockManager'
+
+const router = express.Router({ mergeParams: true })
+const web3Service = new Web3Service(networks)
+
+const verifierController = new VerifierController({
+  web3Service,
+})
+
+router.get('/list/:network/:lockAddress', lockManagerMiddleware, (req, res) =>
```

Comment:
You can put middleware on "/" instead and it will apply it to all routes that come after so you won't need to repeat it.

You should also add authenticatedOnly middleware since your API depends on authenticated users.

Reply:
Nice! @kalidiagne fixed this :)